layouts/partials/job/single_header.html

```diff
@@ -1,11 +1,11 @@
-{{ $jobName := (lower (replace .job " " "_")) }}
-<div class="h-60 xl:h-80 bg-no-repeat bg-cover" style='background-image: url(/theme-assets/jobs/{{ lower .role }}/{{ $jobName }}/{{ $jobName }}_background.png);'>
+{{ $jobName := (lower (replace .job "_" " ")) }}
```

Comment:
Wouldn't it be better to get the jobname from [here](https://github.com/The-Balance-FFXIV/glam/blob/ac77d21174ab7ca4a382402f2d8032b46b017a62/layouts/jobs/bis.html#L8)? Or even doing `.Parent.Title`?

Reply:
job (taken from job_name) contains the underscore. The .Parent.Title one could work though, yes. Should I change it to that?

Reply:
Actually, it doesn't seem to work locally...

Reply:
I would add another parameter to the header dict, and just use that – it's more or less guaranteed to be correct in the `balance-static` editor.

Reply:
Of course, this also means updating all the templates, but I don't think that's awful all things considered.

Reply:
it's only a couple of files, ye. I'll go with that then.

Reply:
Adding the parameter and using the .Parent.Title should not introduce any breaking changes, so we should be fine.